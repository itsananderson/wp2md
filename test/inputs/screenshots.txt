== Screenshots ==

1. These should become images
2. With captions
3. This caption should * [escape] the "asterisk" and "square brackets"

3 but this one shouldn't

and neither should this one