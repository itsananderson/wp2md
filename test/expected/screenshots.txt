Screenshots
-----------

![These should become images](screenshot-1.png "These should become images")

*These should become images*

![With captions](screenshot-2.png "With captions")

*With captions*

![This caption should &#42; &#91;escape&#93; the &quot;asterisk&quot; and &quot;square brackets&quot;](screenshot-3.png "This caption should &#42; &#91;escape&#93; the &quot;asterisk&quot; and &quot;square brackets&quot;")

*This caption should &#42; &#91;escape&#93; the &quot;asterisk&quot; and &quot;square brackets&quot;*

3 but this one shouldn't

and neither should this one